controllers/goflowkube/goflowkube_test.go

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
FYI - though I couldn't find any written rule about that, in general we don't write license headers per file, considering the LICENSE file at the project root is sufficient.

Reply:
btw thanks for writing these tests!

Reply:
You are welcome !

I saw that in [/api/v1alpha1/flowcollector_types.go#L1](https://github.com/jpinsonneau/network-observability-operator/blob/14deb157fc3ddd0db59bb68f3be543490b67b2cd/api/v1alpha1/flowcollector_types.go#L1) but not in the other ones.

We can remove them for sure :+1: